packages/payment-processor/test/payment/batch-proxy.test.ts

```diff
@@ -430,16 +478,70 @@ describe('erc20-batch-conversion-proxy', () => {
           expect(
             BigNumber.from(initialDAIFromBalance).sub(BigNumber.from(DAIFromBalance)),
             // Calculation of expectedAmountToPay when there there is no fee USD limit
-            //   expectedAmount:      1.00
-            //   feeAmount:        +   .02
-            //                     =  1.02
+            //   expectedAmount:    55 000
+            //   feeAmount:        +     2
             //   AggEurUsd.sol     x  1.20
             //   AggDaiUsd.sol     /  1.01
             //   BATCH_CONV_FEE      x  1.003
-            //   (exact result)    =  1.215516831683168316 (over 18 decimals for this ERC20)
```

Comment:
The comments were wrong: the request amount and fees were 55000 and 2. 
I removed the one about the exact results, since it was wrong as well. Didn't add a new exact result as I didn't find it brings much value since we already have the detailed calculus